### PR TITLE
Feature/dev 2467 price derivation

### PR DESCRIPTION
**Pull request by lapatric**

### CODE REVIEW

**Review by TaprootFreak — APPROVED** [review]

File: src/components/exchange-rate.tsx [hunk 1]
```diff
@@ -31,6 +48,13 @@ export function ExchangeRate({ exchangeRate, rate, fees, feeCurrency, from, to }
       ? 'Arbitrum / Optimism'
       : undefined);
 
+  const outputInfo =
+    type === 'buy'
+      ? 'The output amount is computed as the input amount minus the DFX fee and the network fee over the base rate. That is, {{output}} {{outputSymbol}} = ({{input}} {{inputSymbol}} - {{dfxFee}} {{feeSymbol}} - {{networkFee}} {{feeSymbol}}) ÷ {{baseRate}}.'
+      : type === 'sell'
+      ? 'The output amount is computed as the input amount times the base rate minus the DFX fee and the network fee. That is, {{output}} {{inputSymbol}} = {{input}} {{outputSymbol}} × {{baseRate}} - {{dfxFee}} {{feeSymbol}} - {{networkFee}} {{feeSymbol}}.'
+      : 'The output amount is computed as the input amount minus the DFX fee and the network fee over the base rate. That is, {{output}} {{inputSymbol}} = ({{input}} {{outputSymbol}} - {{dfxFee}} {{feeSymbol}} - {{networkFee}} {{feeSymbol}}) × {{baseRate}}.';
```
Comment (davidleomay) [inline src/components/exchange-rate.tsx:56]: Translations for buy, sell and swap look very similar. Can we combine them?
Shouldn't it be `÷ {{baseRate}}` for buy and swap?

Comment (davidleomay) [reply src/components/exchange-rate.tsx:56]: Is this correct? `{{output}} {{inputSymbol}} = ...`

Comment (davidleomay) [reply src/components/exchange-rate.tsx:56]: You probably don't need the type parameter, you can simply check whether the fee currency matches the input or output currency.

File: src/screens/transaction.screen.tsx [hunk 2]
```diff
@@ -390,11 +390,33 @@ function TxInfo({ tx }: TxInfoProps): JSX.Element {
     (p) => p !== CryptoPaymentMethod.CRYPTO,
   ) as FiatPaymentMethod;
 
+  const exchangeRateInfo = translate(
+    `screens/payment`,
+    tx.type === TransactionType.BUY
+      ? 'Received amount = (Input amount - DFX fee - Network fee) ÷ Base rate.'
+      : tx.type === TransactionType.SELL
+      ? 'Output amount = Input amount × Base rate - DFX fee - Network fee.'
+      : 'Output amount = (Input amount - DFX fee - Network fee) × Base rate.',
```
Comment (davidleomay) [inline src/screens/transaction.screen.tsx:399]: Why received vs. output amount?

**Review by davidleomay — APPROVED** [review]

**Pull request merged**